src/main/java/com/coumin/woowahancoupons/store/service/SimpleStoreService.java

```diff
@@ -1,8 +1,27 @@
 package com.coumin.woowahancoupons.store.service;
 
+import com.coumin.woowahancoupons.domain.coupon.CouponRepository;
+import java.util.List;
+import java.util.stream.Collectors;
 import org.springframework.stereotype.Service;
+import com.coumin.woowahancoupons.coupon.dto.StoreCouponResponseDto;
+import org.springframework.transaction.annotation.Transactional;
 
+@Transactional(readOnly = true)
 @Service
 public class SimpleStoreService implements StoreService {
 
+	private final CouponRepository couponRepository;
+
+	public SimpleStoreService(
+		CouponRepository couponRepository) {
+		this.couponRepository = couponRepository;
+	}
```

Comment:
public SimpleStoreService(CouponRepository couponRepository) {
		this.couponRepository = couponRepository;
	}